zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/ParagraphTest.java

```diff
@@ -70,6 +70,18 @@ public void scriptBodyWithReplName() {
 
     text = "%table 1234567";
     assertEquals("1234567", Paragraph.getScriptBody(text));
+
+    text = "%presto (testPropertyKey) 1234567";
+    assertEquals("(testPropertyKey) 1234567", Paragraph.getScriptBody(text));
+
+    text = "%spark (1234567)";
+    assertEquals("(1234567)", Paragraph.getScriptBody(text));
+
+    text = "%jdbc(presto) (testPropertyKey) 1234567";
+    assertEquals("(testPropertyKey) 1234567", Paragraph.getScriptBody(text));
+
+    text = "%jdbc(presto) (testPropertyKey) (select 1234567)";
+    assertEquals("(testPropertyKey) (select 1234567)", Paragraph.getScriptBody(text));
```

Comment:
I think "presto" would be treated as the replName which should be stripped.

There was a syntax change from %jdbc(replName) to %replName. So replName is figured out first then by looking at interpreter configs, it figures out that replName belong to jdbc group.

My change uses regex, and would also make the old syntax work in the new version.